docs/integration_resources/resources/usd_tutorials/usd_tutorial_01.md

```diff
@@ -0,0 +1,60 @@
+## Creating Your First USD Stage
+
+This tutorial is focused on creating, using the operators from Shift_USD catalog, a simple USD stage containing a transform and a sphere.
+
+In order to follow easily the steps, it is recommended to read the documentation about the [Shift's interface](../../../getting_started/basics/ui_overview.md) and [workflow execution](../../../getting_started/basics/execute.md).
+
+
+### Tutorial
+
+1. Open Shift application. A new window's instance of Shift should appear like the following one:
```

Comment:
I would mention somewhere to check for the USD operators in the Node list . If not, redirect the user to the Shift_USD installation page.

Reply:
Okay, let me know your opinion on the change and I can copy it to the rest of the tutorials 👍